common.mk

```diff
@@ -5,26 +5,33 @@ ifndef common_dir
 endif
 
 build = $(common_dir)/build.sh
+test-openshift = $(common_dir)/test-openshift.sh
 
 ifeq ($(TARGET),rhel7)
 	OS := rhel7
 else
 	OS := centos7
 endif
 
-script_env = \
+build_env = \
 	SKIP_SQUASH=$(SKIP_SQUASH)                      \
-	UPDATE_BASE=$(UPDATE_BASE)                      \
+	UPDATE_BASE=$(UPDATE_BASE)
+
+script_env = \
+	VERSION="$(VERSION)"                            \
 	VERSIONS="$(VERSIONS)"                          \
 	OS=$(OS)                                        \
-	VERSION="$(VERSION)"                            \
```

Comment:
I'm curious why VERSION was moved above;  and while we are on it .. is the VERSION && VERSIONS needed, or is that just leftover?